freshmaker/pyxis_gql.py

```diff
@@ -174,7 +176,11 @@ def find_repositories(self, published=None, release_categories=None, auto_rebuil
             result = self.query(query_dsl)
             error = result["find_repositories"]["error"]
             if error is not None:
-                raise PyxisGQLRequestError(str(error))
+                trace_msg = ""
```

Comment:
`trace_id` is not in Graphql returned data, it's in response header:
https://gql.readthedocs.io/en/v3.3.0/usage/headers.html

Instead of logging this in all methods, you can do it in `query`: https://github.com/redhat-exd-rebuilds/freshmaker/blob/9788db751ee6c94c6b59c01464622c53c4bdc494/freshmaker/pyxis_gql.py#L65 .

Reply:
I see! I'll correct that. I also liked the idea of logging in the `query` method, instead of in all methods, thanks a lot!